Astro CLI v1.0.0 (All Changes) (#690)
astronomer/docs: software/upgrade-to-0-28.md

Make changes as needed and rerun the upgrade command from Step 6. Do not continue to Step 8 until you have successfully created a new Airflow Deployment.

## Step 8: Upgrade the Astro CLI to v0.28

To ensure reliability and full access to features included in Astronomer Software v0.28, all users must upgrade to v0.28 of the Astro CLI. We recommend the latest available version, though you can install any patch release within the v0.28 series.

To upgrade to the latest available v0.28 version of the Astro CLI, run:

`curl -sSL https://install.astronomer.io | sudo bash -s -- v0.28`

To do so via Homebrew, run:

`brew install astronomer/tap/astro@0.28`

> Earlier versions of the Astro CLI are backwards incompatible with Astronomer v0.28. All team members within your organization must upgrade the Astro CLI individually before taking any further action on the platform or in a local Airflow environment. For a detailed breakdown of CLI changes between versions, refer to the [CLI Release Notes](cli-release-notes.md). For detailed install guidelines and more information on the Astro CLI, refer to [Astro CLI Quickstart](cli-quickstart.md).

Reviewer: ```suggestion
Earlier versions of the Astro CLI are backwards incompatible with Astronomer v0.28. All team members within your organization must upgrade the Astro CLI individually before taking any further action on the platform or in a local Airflow environment. For more information about CLI changes between versions, see [CLI Release Notes](cli-release-notes.md). For installation guidelines and information about the Astro CLI, see [Astro CLI Quickstart](cli-quickstart.md).
```
